traits/trait_list_object.py

```diff
@@ -416,6 +416,23 @@ def rename(self, name):
             self.name = name
             self.trait = trait.handler
 
+    def clear(self):
+        if self.trait.minlen > 0:
+            self.len_error(0)
+
+        removed = list.copy(self)
+        list.clear(self)
+        index = 0
+
+        self._send_trait_items_event(
+            self.name_items, TraitListEvent(index, removed)
+        )
+
+    def copy(self):
```

Comment:
So in discussion with @corranwebster, we realised that actually there's no need for a `copy` method at all; we should just inherit the `list.copy` base class method.

In particular, `copy` on a `List` value should return a plain old Python list.

Suggest removing this method, but keeping the tests, and adding an additional test to verify that the result of the `copy` is a `list`, not a `TraitListObject`.

Reply:
Thanks, I removed the `copy` method and added the test.